fe/fe-core/src/main/java/com/starrocks/scheduler/mv/MVEpoch.java

```diff
@@ -49,17 +60,41 @@ public class MVEpoch implements Writable {
     @SerializedName("commitTimeMilli")
     private long commitTimeMilli;
 
+    @SerializedName("commitInfos")
+    private List<TabletCommitInfo> commitInfos = new ArrayList<>();
```

Comment:
Why persist it ? Why not just link the txnId here ?

Reply:
fixed. I havn't get the persisted logic here yet.